app/src/test/kotlin/eu/quiqua/gloomhaven/statstracker/app/ApplicationSpek.kt

```diff
@@ -0,0 +1,19 @@
+package eu.quiqua.gloomhaven.statstracker.app
+
+import com.natpryce.hamkrest.assertion.assert
+import com.natpryce.hamkrest.present
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+import org.jetbrains.spek.api.dsl.on
+
+internal class ApplicationSpek : Spek({
+    describe("An application is created") {
+        on("create") {
+            val application = Application()
+            it("provides components via dagger") {
+                assert.that(application.applicationComponent.context, present())
```

Comment:
I am not sure if the `present` matcher is invoked correctly here 🙈 
`hamkrest` is a bit confusion as well :D